common/types/block.go

```diff
@@ -128,8 +128,15 @@ type BlockHeader struct {
 	Data             []byte
 	Coin             bool
 	Timestamp        int64
-	BlockVotes       []BlockID
-	ViewEdges        []BlockID
+
+	BaseBlock BlockID
+
+	AgainstDiff []BlockID
+	ForDiff     []BlockID
+	NeutralDiff []BlockID
+
+	BlockVotes []BlockID
```

Comment:
is this still necessary?

Reply:
if we want to keep the original tortoise code and it's tests then yes. currently they are just empty on the wire so not much to worry about. we can remove them and the tortoise all together..

Reply:
if you think we should keep it then its ok